src/avatar-selector.html

```diff
@@ -6,9 +6,9 @@
   <title>avatar selector</title>
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <% if(NODE_ENV === "production") { %>
```

Comment:
FYI: @gfodor has PR #217, which was originally to bump A-Frame to `0.8.2`: https://github.com/mozilla/hubs/pull/217/files

as I understand it, it's probably best to point to `3e7a4b3`. likely, PR #217 can be closed?